### ISSUE TITLE
Killed clustering

### ISSUE THREAD

**Issue by EllistonV** [mothur/mothur.github.io#80]
Hello,

So I tried to run `cluster.split`. It took a lot of time for it to calculate the distances. Also, the dist files were big (+250 GB big). When it was doing the clustering, I got this message:

```
cluster.split(fasta=final.fasta, count=final.count_table, taxonomy=final.taxonomy, taxlevel=4, cutoff=0.03)
/******************************************/
/******************************************/
Selecting sequences for group Sphingomonadales (2 of 60)
Number of unique sequences: 36453

...

Output File Names:
final.10.dist

Clustering final.3.disttemp

tp	tn	fp	fn	sensitivity	specificity	ppv	npv	fdr	accuracy	mcc	f1score
6.60748e+07	1.47227e+08	8.81038e+06	5.46799e+06	0.92357	0.943537	0.882348	0.96419	0.882348	0.93726	0.856761	0.902489	

Killed
```

I tried it the "trad" way using `dist.seqs` and then `cluster`. Again, the dist file was very big and I got the following message after running `cluster` :

```
mothur > dist.seqs(fasta=final.fasta, cutoff=0.03)

...


It took 8414 secs to find distances for 164436 sequences. 2045548065 distances below cutoff 0.03.

Output File Names: 
final.dist

mothur > cluster(column=final.dist, count=final.count_table, cutoff = 0.03)


Using 12 processors.

Clustering final.dist
iter	time	label	num_otus	cutoff	tp	tn	fp	fn	sensitivity	specificity	ppv	npv	fdr	accuracy	mcc	f1score

0.03
Killed
```

This is the `summary.seqs` before clustering:
```

		Start	End	NBases	Ambigs	Polymer	NumSeqs
Minimum:	1	472	227	0	3	1
2.5%-tile:	1	472	252	0	4	9546
25%-tile:	1	472	253	0	5	95457
Median: 	1	472	253	0	5	190914
75%-tile:	1	472	253	0	5	286370
97.5%-tile:	1	472	253	0	5	372281
Maximum:	1	472	259	0	8	381826
Mean:	1	472	252	0	4
# of unique seqs:	164436
total # of seqs:	381826

```

I have never had this problem with the training files from the workshop and with real data from a labmate (which had many much more sequences). I don't know if there are too many unique sequences, or if there is a problem with my sequences. 
My computer has 24 GB of RAM (I know, I need to get another 16GB stick) and was using 12 cores at 2.0 GHz speed.

In advance, thank you!

**Comment by pschloss** [comment]
Hi - do you have 2x150 or 2x250 nt reads? If it's 2x150 then I suspect you have a high error rate that is inflating the number of unique sequences and causing problems.

You might try `cluster.split` with taxlevel=5 or 6.

You might also try `cluster.split` in two steps where you use 12 processors on the splitting and distance calculation step and 1 processor on the clustering step.

These suggestions should help with the memory limitation issues.

Pat

**Comment by EllistonV** [comment]
Hello,

I have 2x250. 

I ran `cluster.splits` with a taxlevel=6, but I had the same problem. The group that is causing so many problems is Rhizobials, almost all sequences are from that group (67,000 sequences just for that group? which doesn't make sense for my samples anyway). I think I just got a bad sequencing run.

Is there a way to run `summary.qual` in all the qual files made with `fastq.info`?
